daemon/daemon_unix.go

```diff
@@ -159,6 +159,21 @@ func getBlkioWeightDevices(config containertypes.Resources) ([]specs.WeightDevic
 	return blkioWeightDevices, nil
 }
 
+func parseAndSetSecurityOpt(daemon *Daemon, container *container.Container, config *containertypes.HostConfig) error {
+	var findNNP bool
+	err := parseSecurityOpt(container, config)
```

Comment:
it seems like youd want to return the error early here instead of iterating thru all the opts then returning the error? or maybe add a comment as to why this is ok